mono_repo/lib/src/commands/github/github_yaml.dart

```diff
@@ -373,6 +373,9 @@ class _CommandEntry {
         'name': name,
         if (env != null && env.isNotEmpty) 'env': env,
         'run': run,
+        // We want to run even if other steps failed, use stages if you want
+        // to block things on previous failures.
+        'if': 'always()',
```

Comment:
@jakemac53 – but the Job will still fail if one of the steps fails, right?

Reply:
Ooo! https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#steps-context

Reply:
Looks like it does (there are failures now)